packages/libmuparser/build.sh

```diff
@@ -0,0 +1,13 @@
+TERMUX_PKG_HOMEPAGE="https://beltoforion.de/en/muparser"
+TERMUX_PKG_DESCRIPTION="An extensible high performance math expression parser \
+library written in C++"
+TERMUX_PKG_GROUPS="science"
+TERMUX_PKG_LICENSE="BSD 2-Clause"
+TERMUX_PKG_LICENSE_FILE="LICENSE"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="2.3.3-1"
+TERMUX_PKG_SRCURL="https://github.com/beltoforion/muparser/archive/refs/tags/v$TERMUX_PKG_VERSION.tar.gz"
+TERMUX_PKG_SHA256=91d26d8274ae9cd9c776ee58250aeddc6b574f369eafd03b25045b858a2b8177
+TERMUX_PKG_BUILD_DEPENDS="cmake"
```

Comment:
This line is not needed, cmake is installed by build system automatically. `TERMUX_PKG_BUILD_DEPENDS` means that it downloads cmake that has been cross-compiled for termux, and that cmake cannot be used in any case